### PR TITLE
Support Openai API

### PR DESCRIPTION
**Pull request by chenqianhe**

### CODE REVIEW

**Comment by chenqianhe** [comment]
#102

**Comment by chenqianhe** [comment]
#73

**Comment by chenqianhe** [comment]
# Test List:
- 原始功能均无影响：
    - local 转录
    - cut（测试输出文本文件是有异常进而导致cut 异常）
    - 检测文件夹自动转录
    - 合并音视频
- Openai API 方式支持所有功能正常调用：
    - 转录
    - cut（测试输出文本文件是有异常进而导致cut 异常）
    - 检测文件夹自动转录
    - 合并音视频

**Comment by chenqianhe** [comment]
可优化：openai 转录部分频繁文件读写耗时

**Comment by chenqianhe** [comment]
> 可优化：openai 转录部分频繁文件读写耗时

multiprocess

File: autocut/utils.py [hunk 1]
```diff
@@ -2,8 +2,23 @@
...
 import re
 
-import srt
+import ffmpeg
+import numpy as np
 import opencc
+import srt
+
+
+def load_audio(file: str, sr: int = 16000) -> np.ndarray:
```
Comment (suanmiao) [inline autocut/utils.py:11]: This step can be quite expensive and memory intensive? Is it possible to directly give the file to local/openai whisper? both of them should have that option?

**Review by suanmiao — COMMENTED** [review]
Left some comments, the main comment is about supporting openai-whisper as a model instead of mode, since there is only one model for openai whisper, and also a comment about avoiding loading audio into ndarray since it can introduce unnecessary memory consumption.

File: autocut/type.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+    "Thai",
+    "Turkish",
+    "Ukrainian",
+    "Urdu",
+    "Vietnamese",
+    "Welsh",
+]
+
+
+class WhisperModel(Enum):
```
[the remark this answers is not in the document]
Comment (chenqianhe) [reply autocut/type.py:67]: there will be other ways to run whisper, like fast-whisper, whisper.cpp, etc.

File: autocut/utils.py (hunk 1 above)
Comment (chenqianhe) [reply autocut/utils.py:11]: Vad is a pre selectable step for all model inference, requiring the reading of audio files

**Pull request merged**